client/admin/components/users/ImportDialog.vue

```diff
@@ -0,0 +1,171 @@
+<template>
+  <v-dialog v-hotkey="{ esc: close }" v-model="visible" width="700">
+    <v-btn slot="activator" color="blue-grey" outline>
+      <v-icon>cloud_upload</v-icon>Import
+    </v-btn>
+    <v-form @submit.prevent="save">
+      <v-card class="pa-3">
+        <v-card-title class="headline">
+          Import Users
+        </v-card-title>
+        <v-card-text>
+          <v-text-field
+            ref="textFile"
+            v-model="filename"
+            :error-messages="showErrors"
+            :disabled="importing"
+            @click="$refs.fileInput.click()"
+            prepend-icon="attach_file"
+            label="Upload .xls, .xlsx or .csv file"
+            readonly
+            single-line/>
+          <input
+            v-validate="rules"
+            v-show="false"
+            ref="fileInput"
+            @change="onFileSelected"
+            data-vv-name="file"
+            type="file"/>
```

Comment:
Don't _selfclose_ html tags. It's not required and makes reader think that they are looking at vue component instead :bowtie: